lyrics-core/src/main/java/com/flipkart/lyrics/helper/Injector.java

```diff
@@ -68,4 +68,11 @@ public static void processTypeAdditionalHandlers(Tune tune, MetaInfo metaInfo) {
             h.setMetaInfo(metaInfo);
         });
     }
+
+    public static void processFieldModificationHandlers(Tune tune, MetaInfo metaInfo) {
+        tune.getFieldModificationHandlers().values().forEach(h -> {
+            h.setTune(tune);
+            h.setMetaInfo(metaInfo);
```

Comment:
All handlers can implement a common interface wherein we can inject tune and metainfo. We don't need to keep adding these methods for each extension of library.